### DISCUSSION TITLE
Aqara Temperature and Humidity - WSDCGQ12LM - Reporting Frequency

### DISCUSSION THREAD

**Discussion by maptz** [Koenkk/zigbee2mqtt#23112] [Devices]
Hi, 

I've got a pair of Aqara WSDCGQ12LM devices. I had it setup so that it was reporting temperature and pressure every minute, but I had a power cut and now I've had to reboot the Zigbee2mqtt system. 

For some reason, now the reporting is rather random from each device (around every 5 minutes). I can't seem to update the Reporting frequency in Zigbee2MQTT. It fails with 

```
Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'Bind 0x54ef441000a56f4b/1 msTemperatureMeasurement from '0x6c5cb1fffefc7f61/1' failed ({"address":39334,"clusterId":32801,"sequence":31} after 10000ms)'
```

I'm sure I successfully set this once, but I don't seem to be able to repeat it. 

Can anyone tell me how to set the reporting frequency of the device?

**Comment by francisp2** [comment]
It is a battery device, so it needs to be awake where changing settings. Press the button just before you change the settings.

**Comment by maptz** [comment]
Amazing @francisp2. You're totally correct. This worked a treat. 

For the sake of any future googling, here's what worked for me:

To set the Aqara temperature modules to report more frequently, you must go to the Reporting page in Zigbee2MQTT

- Select Endpoint 1
- Set Cluster to Temperature
- Set Attribute to MeasuredValue
- Set Min rep interval to 60
- Set Max rep interval to 3600
- Set Min rep change 0

Now. **Very important** You must press the button on the module itself just before pressing apply on the web interface.

**Reply by KatyPurry** [reply]
> Amazing @francisp2. You're totally correct. This worked a treat.
> 
> For the sake of any future googling, here's what worked for me:
> 
> To set the Aqara temperature modules to report more frequently, you must go to the Reporting page in Zigbee2MQTT
> 
> * Select Endpoint 1
> * Set Cluster to Temperature
> * Set Attribute to MeasuredValue
> * Set Min rep interval to 60
> * Set Max rep interval to 3600
> * Set Min rep change 0
> 
> Now. **Very important** You must press the button on the module itself just before pressing apply on the web interface.

Sorry to jump in on a closed topic but I came across this while researching how I could change my update frequency for my aqara sensor. 

I'm fairly new to HA/Z2M, but I did understand that I had to go to the Reporting tab. For example, I've been able to change the update frequency for my electrical outlets. 

However, by default for the Aqara sensor, this section is empty. I choose 1 for Endpoint and Temperature for Cluster, but the Attribute field is not a drop-down list. How do you know to set "MeasuredValue"? I looked on the product sheet on Zigbee2mqtt.io but this info doesn't appear. 

Thanks a lot!

**Reply by nicola-spreafico** [reply]
@KatyPurry  Don't you have this dropdown?
![image](https://github.com/user-attachments/assets/bb315465-ee11-4d29-93ab-77d2495d1dc2)
